front/pages/api/poke/workspaces/[wId]/data_sources/[dsId]/config.ts

```diff
@@ -23,9 +23,8 @@ async function handler(
     session,
     req.query.wId as string
   );
-  const owner = auth.workspace();
```

Comment:
just to confirm: this is thanks to the new auth wrapper we use on endpoints?

Reply:
yes